sphinx_js/typedoc.py

```diff
@@ -68,6 +68,25 @@ def _containing_module(self, node) -> Optional[Pathname]:
                 # Found one!
                 return Pathname(make_path_segments(node, self._base_dir))
 
+    def _containing_deppath(self, node) -> Optional[Pathname]:
+        """Return the path pointing to the module containing the given node.
+        The path is absolute or relative to `root_for_relative_js_paths`.
+        Raises ValueError if one isn't found.
+
+        """
+        while True:
```

Comment:
This has some code duplication with `_containing_module()`. Maybe that can be extracted to an iterator that `_containing_module()` and `_containing_deppath()` can use